themes/digital.gov/src/scss/new/_footnote.scss

```diff
@@ -0,0 +1,7 @@
+@use "uswds-core" as *;
+
+.dg-footnote {
+  li {
+    overflow-wrap: anywhere;
```

Comment:
**question:** Have we tested this with nested lists?

Reply:
Tested with nested lists, confirming that it works with nested lists.

**Single List**
<img width="506" alt="single_list" src="https://github.com/user-attachments/assets/08f8b9f5-dd17-41a8-85d4-9ba7fdecd22e">

**Nested List**
<img width="544" alt="nested_list" src="https://github.com/user-attachments/assets/5f7659a8-0165-4307-908a-79172dc90157">